docs/source/gbd2019_models/causes/measles/index.rst

```diff
@@ -0,0 +1,398 @@
+supplementation given to a percentage of the population) that can reduce measles
+incidence as a downstream effect.
+
+In particular, there are various uses for which this model is **not** suitable.
+For example:
+
+1. The simple measles model described here does not explicitly incorporate
+vaccine coverage or efficacy, hence cannot be used to model the impact of a
+vaccination campaign.
+
+2. This model uses country-level data, and cannot be used to model local measles
+outbreaks due to lack of vaccination in small communities.
+
+Some of the **assumptions** made in this model are:
+
+1. There is no data available for population in recovered state in GBD. Since the early and late neonatal age groups are not modeled in GBD, we made the assumption that there are no individuals in the recovered state at the start of the post-neonatal age group. We then calculated the proportion of the population in the recovered state in the 1-4 year age group using GBD measles incidence and mortality rates. Note that we performed this calculation only for the 1-4 age group because measles is most often of interest among the children under five population, but this assumption could be expanded to other older ages as well.
+
+2. There is no data avaialable for remission rate in GBD. So a constant remission rate is 
+calculated from average case duration assumption of 10 days [GBD-2017-YLD-Capstone-Appendix-1]_.
+
+
+.. list-table:: Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     - 
+   * - Female only
+     - False
+     - 
+   * - YLL only
+     - False
+     - 
+   * - YLD only
+     - False
+     - 
+   * - YLL age group start
+     - Post Neonatal
+     - GBD age group id is 4
+   * - YLL age group end
+     - 50 to 54 years
+     - GBD age group id is 15
+   * - YLD age group start
+     - Post Neonatal
+     - GBD age group id is 4
+   * - YLD age group end
+     - 50 to 54 years
+     - GBD age group id is 
+
+Notably, our cause model as described accounts for the number of simulants that enter the recovered state by becoming infected with measles and recovering, but does not consider simulants who enter the recovered state by receiving the measles vaccine without becoming infected with measles. Therefore, we are underestimating the prevalence of the recovered state in our model. This strategy was employed so that the size of the susceptible and recovered populations at model initialization will remain stable as the simulation runs and simulants progress through the SIR cause model (this is an improvement from our previous assumption that there was zero prevalence of the recovered state upon model initialization, which caused an decrease in the susceptible population and an increase in the recovered population over time, which affected calibration of simulation incidence and mortality rates to GBD rates).
+
```

Comment:
Maybe somewhere in here, add a little more clarification why it wasn't stable. something like:

incidence_gbd = #measles/(S+I+R)

In vivarium, we apply incidence to the S population only, so we must adjust the incidence_gbd appropriately so that we may get the correct number of measles cases 

incidence without prevR adjustment - the denominator is 1- prevI:  
 = #measles/(S+R)

incidence with prevR adjustment - the denominator is 1-prevI-preR:
 =  #meales/S

Without the prevR adjustment, the denominator is bigger, therefore the incidence is smaller, so we will underestimate measles cases. Hence, as we apply an under-estimated incidence rate to a shrinking S population, we get lower measles CSMR over time. 

~ something like that??